packages/summer-earn-protocol-subgraph/src/client.ts

```diff
@@ -0,0 +1,4109 @@
+// @ts-nocheck
+// This file was automatically generated and should not be edited.
+import { GraphQLClient, RequestOptions } from 'graphql-request';
+import gql from 'graphql-tag';
```

Comment:
that's missing dep in the pkg.json

Reply:
not sure if it's required ?
the client is autogenerated and the dependency is the dependency of library that generates the client.

btw not sure why the autogenerated client is not importing `gql` directly from `graphql-request`

Reply:
if autogenerated then should be good